clients/cobol-lsp-vscode-extension/src/test/suite/processorgroups.spec.test.ts

```diff
@@ -69,7 +68,7 @@ suite("Integration Test Suite: Processor Groups", function () {
     const diagPromise = helper.waitForDiagnosticsChange(extSrcPath);
     await helper.showDocument(extSrcPath);
     const diagnostics = await diagPromise;
-    assert.ok(diagnostics.length > 0);
+    assert.ok(diagnostics.length = 1);
```

Comment:
Well, this is certainly wrong... (and I can't believe that it can be done)
```suggestion
    assert.ok(diagnostics.length === 1);
```

Reply:
🤦